File: specification/sql/resource-manager/Microsoft.Sql/SQL/stable/2025-01-01/examples-js/ManagedInstanceGetWhileUpdating.js
const { SqlManagementClient } = require("@azure/arm-sql");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to gets a managed instance.
 *
 * @summary gets a managed instance.
 * x-ms-original-file: 2025-01-01/ManagedInstanceGetWhileUpdating.json
 */
async function getManagedInstanceWhileResourceIsUpdating() {
  const credential = new DefaultAzureCredential();
  const subscriptionId = "00000000-1111-2222-3333-444444444444";
  const client = new SqlManagementClient(credential, subscriptionId);
  const result = await client.managedInstances.get("testrg", "testinstance");
  console.log(result);
}

File: specification/sql/resource-manager/Microsoft.Sql/SQL/preview/2026-08-01-preview/examples-java/ManagedInstanceGetWhileUpdating.java

/**
 * Samples for ManagedInstances GetByResourceGroup.
 */
public final class Main {
    /*
     * x-ms-original-file: 2026-08-01-preview/ManagedInstanceGetWhileUpdating.json
     */
    /**
     * Sample code: Get managed instance while resource is updating.
     * 
     * @param manager Entry point to SqlServerManager.
     */
    public static void
        getManagedInstanceWhileResourceIsUpdating(com.azure.resourcemanager.sql.SqlServerManager manager) {
        manager.serviceClient().getManagedInstances().getByResourceGroupWithResponse("testrg", "testinstance", null,
            com.azure.core.util.Context.NONE);
    }
}
